.concourse/validate-pr.yaml

```diff
@@ -0,0 +1,77 @@
+---
+
+# s3, git, time, github-release, registry-image,
+# pool - serialize access to external systems
+# git - can also push
+
+resource_types:
+  - name: pull-request
+    type: docker-image
+    source:
+      repository: teliaoss/github-pr-resource
+
+resources:
+#  - name: istio-installer #DO NOT SET TO FORK - for testing use another pipeline and pause this one
+#    type: git
+#    source:
+#      uri: "https://github.com/istio/istio-installer"
+#
+  - name: istio-master
+    type: git
+    source:
+      uri: "https://github.com/istio/istio"
+      branch: master
+
+  - name: istio-installer-pr
+    type: pull-request
+    source:
+        repository: "istio/istio-installer"
+        access_token: ((GITHUB_ACCESS_TOKEN))
+
+jobs:
+  - name: validate-istio-installer-pr
+    serial_groups: [ cluster ]
+    plan:
+    - get: istio-installer-pr
+      trigger: true
+    - get: istio-master
+      trigger: false
+    - put: istio-installer-pr
+      params:
+        path: istio-installer-pr
+        status: pending
+    - task: validate
+      params:
+        KUBECONFIG: ((KUBECONFIG))
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: gcr.io/peripli/istio-base
```

Comment:
Should we really use an image from Peripli within istio? Maybe we rather use an istio image or push a new one if there is no suitable one.